bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/target/env/Asterism.scala

```diff
@@ -190,4 +194,71 @@ object Asterism {
   def createSingleAsterism: Single = {
     Single(new SPTarget())
   }
+
+  def fromTypeAndTemplateTarget(
+    asterismType: AsterismType,
+    target:       SPTarget
+  ): Asterism = {
```

Comment:
We have to create a proper GHOST asterism but have only one target from Phase 1.  This method compensates by adding and subtracting from the P1 target.  Presumably the PI will fill in the proper values during the phase 2 process after instantiation.

Reply:
Would it make sense to generate a second target for dual-target proposal to a given offset to the one in the proposal?

Reply:
Oh, I see you're already doing that